terraform/modules/cloud/aws/deploy/vpc/main.tf

```diff
@@ -0,0 +1,39 @@
+resource "aws_vpc" "vpc" {
+  cidr_block = "10.0.0.0/16"
+}
+
+data "aws_availability_zones" "available" {}
+
+resource "aws_subnet" "subnet" {
+  vpc_id                  = "${aws_vpc.vpc.id}"
+  count                   = "${length(data.aws_availability_zones.available.names)}"
+  cidr_block              = "10.0.${count.index}.0/24"
```

Comment:
I thought there are better TF functions for sub-netting :)

Reply:
nope :D